BusinessTrip.java

```diff
@@ -0,0 +1,68 @@
+package by.gsu.epamlab;
+
+public class BusinessTrip {
+
+    private final static int RATE = 10000;
+    private String account;
+    private int transport;
+    private int days;
+
+    public BusinessTrip() {
+    }
+
+    public BusinessTrip(String account, int transport, int days) {
+        this.account = account;
+        this.transport = transport;
+        this.days = days;
+    }
+
+    public String getAccount() {
+        return account;
+    }
+
+    public void setAccount(String account) {
+        this.account = account;
+    }
+
+    public int getTransport() {
+        return transport;
+    }
+
+    public void setTransport(int transport) {
+        this.transport = transport;
+    }
+
+    public int getDays() {
+        return days;
+    }
+
+    public void setDays(int days) {
+        this.days = days;
+    }
+
+    public int getTotal(){
+        return transport + RATE*days;
+    }
+
+    private String toFinValue(int value){
+        if (value % 100 == 0)
+            return value/100 + ".00";
+        else if (value % 100 < 10)
+            return value/100 + ".0" + value % 100;
+        else
+            return value/100 + "." + value % 100;
+    }

```

Comment:
It can be solved with regular expressions:
%d - key for whole numbers
%2d - key with amount of elements: if int a = 1, ("%2d", a) -> " 1"
%02d - spaces will be filled with 0: ("%04d", a) -> "0001"